website/src/reusables/_supported-networks.mdx

```diff
@@ -1,3 +1,6 @@
+<!--Content in this file is duplicated into multiple files. 
```

Comment:
As messaged in slack this is not the issue, you can remove this and the other comments in the reusables file

Reply:
Sure.